packages/file_selector/file_selector_platform_interface/lib/src/platform_interface/file_selector_interface.dart

```diff
@@ -74,4 +74,13 @@ abstract class FileSelectorPlatform extends PlatformInterface {
   }) {
     throw UnimplementedError('getDirectoryPath() has not been implemented.');
   }
+
+  /// Open file dialog for loading directories and return multiple directories paths
+  /// Returns `null` if user cancels the operation.
+  Future<List<String>?> getDirectoriesPaths({
+    String? initialDirectory,
+    String? confirmButtonText,
+  }) {
+    throw UnimplementedError('getDirectoriesPaths() has not been implemented.');
+  }
```

Comment:
This new method should have a unit test that it throws unimplemented. Testing changes to the interface is more important than testing changes to the default method channel implementation, because the latter is no longer actually used.